viso_feature_tracking_ros_tool/src/viso_feature_tracking_node/viso_feature_tracking_interface.cpp

```diff
@@ -62,106 +57,20 @@ void VisoFeatureTrackingInterface::process(const sensor_msgs::ImageConstPtr& inp
                                                           << cv_bridge_ptr->image.cols
                                                           << "\nscaling factor="
                                                           << config.scale_factor);
-
-    if (config.scale_factor != 1.0) {
-        cv::Size imageSize = cv_bridge_ptr->image.size();
-        cv::resize(cv_bridge_ptr->image,
-                   cv_bridge_ptr->image,
-                   cv::Size(imageSize.width * config.scale_factor, imageSize.height * config.scale_factor));
-    }
-
-    cv::GaussianBlur(cv_bridge_ptr->image,
-                     cv_bridge_ptr->image,
-                     cv::Size(viso_params.blur_size, viso_params.blur_size),
-                     viso_params.blur_sigma);
-
-    tracker->pushBack(cv_bridge_ptr->image);
-    tracker->getTracklets(this->tracklets_, 0);
-
-    this->timestamps_.push_front(input->header.stamp);
+    tracker_module_->process(cv_bridge_ptr->image, input->header.stamp);
```

Comment:
This is the main part I took out from the original code. All the actual processing happens in the tracker_module_.

Reply:
Great!